manifests/config.pp

```diff
@@ -17,6 +17,15 @@
     ensure_newline => true,
   }
 
+  if $redis::unixsocket != '' {
```

Comment:
What would be configuring this given we only `include redis` ?

Reply:
Technically it could have been changed via hiera. Outside of the installer this can also happen directly via manifests.